packages/remote-config/package.json

```diff
@@ -50,6 +50,7 @@
   "devDependencies": {
     "@firebase/app": "0.13.1",
     "rollup": "2.79.2",
+    "rollup-plugin-dts": "6.1.0",
```

Comment:
Can you change this version to the same as the other versions in this repo? I see that the analytics one already has it set to that version (5.3.1). The result should be that there should be no changes to `yarn.lock`.